src/support/command-line.cpp

```diff
@@ -18,25 +18,52 @@
 
 using namespace wasm;
 
+#ifndef SCREEN_WIDTH
+#define SCREEN_WIDTH 80
+#endif
+
+void printWrap(int leftPad, const std::string &content) {
```

Comment:
perhaps `printWrap` could get a `std::ostream&` and write to that, instead of always writing to `std::cerr`?